### DISCUSSION TITLE
[Miscellaneous] Error Adding Wallet in Superchain Dev Console

### DISCUSSION THREAD

**Discussion by dry-tortuga** [ethereum-optimism/developers#431] [Miscellaneous]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I'm trying to link my wallet in the Superchain Dev console at https://console.optimism.io/, but each time I try to connect and sign the message using my ledger nano s via coinbase wallet (chrome) I see a 422 error with "Invalid SIWE message and/or signature".

I've also tried manually signing the message with etherscan, but it was still rejected with the same error.

I'm able to use Coinbase wallet to connect and sign-in to other sites, so not sure exactly how to troubleshoot this?

I'm using the latest version of the firmware for ledger and i'm on linux 22.0.4.

**Comment by opfocus** [comment]
CC @sbvegan

**Reply by sbvegan** [reply]
I'll flag this one to the team

**Reply by dry-tortuga** [reply]
Thank you!
